### PR TITLE
Use native KeyboardAvoidingView component everywhere

### PR DESCRIPTION
**Pull request by tgolen**
There are a couple of extra things that I threw into this PR that don't need any testing.

- I renamed the FAB to "FloatingActionButton" so that it's clear what it is.
- I increased the max cache key count which helps reliability with large accounts (many reports)

[[Tracking Issue](https://github.com/Expensify/App/issues/10273)]

### Fixed Issues
$ https://github.com/Expensify/App/issues/11118
$ https://github.com/Expensify/App/issues/10905

##### iPad, Android and iOS
⚠️ **DON'T MISS THAT THIS IS TO BE TESTED ON AN iPAD** ⚠️ 
1. _Open_ the app and login with any account
1. _Click_ on the **green ➕ button**
1. _Select_ **New Chat** or **New Group** and then click the "x" to close the modal
2. Verify that the green ➕ is back in it's original position

##### Web / Desktop
Do the same test as above. There won't be any keyboard, so just verify that the page looks correct and there are no weird styles or gaps anywhere.

<details>
<summary>Screenshot iPad</summary>

image

</details>

<details>
<summary>Screenshot iPad mobile web</summary>

image

</details>

<details>
<summary>Screenshot Android native</summary>

image

</details>
<details>
<summary>Screenshot Android mobile web</summary>

image

</details>
<details>
<summary>Screenshot iOS native</summary>

image

</details>
<details>
<summary>Screenshot iOS mobile web</summary>

image

</details>
<details>
<summary>Screenshot Web</summary>

image

</details>
<details>
<summary>Screenshot Desktop</summary>

image

</details>

----------------

##### Android and iOS only
1. Go to a chat that is both pinned and has a draft
6. Tap into the compose box
7. Verify the keyboard does not cover the comment input field

##### Web / Desktop
Do the same test as above. There won't be any keyboard, so just verify that the page looks correct and there are no weird styles or gaps anywhere.

<details>
<summary>Screenshot Android native</summary>

image

</details>
<details>
<summary>Screenshot Android mobile web</summary>

image

</details>
<details>
<summary>Screenshot iOS native</summary>

image

</details>
<details>
<summary>Screenshot iOS mobile web</summary>

image

</details>
<details>
<summary>Screenshot Web</summary>

image

</details>
<details>
<summary>Screenshot Desktop</summary>

image

</details>

----------------

- [x] Verify that no errors appear in the JS console

#### PR Author Checklist
- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] Any functional components have the `displayName` property
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

<details>
<summary><h4>PR Reviewer Checklist</h4>

The reviewer will copy/paste it into a new comment and complete it after the author checklist is completed
</summary>

- [ ] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [ ] I verified the steps for local testing are in the `Tests` section
    - [ ] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] iOS / native
    - [ ] Android / native
    - [ ] iOS / Safari
    - [ ] Android / Chrome
    - [ ] MacOS / Chrome
    - [ ] MacOS / Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [ ] I verified any copy / text that was added to the app is correct English and approved by marketing by tagging the marketing team on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] Any functional components have the `displayName` property
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

</details>

### QA Steps
Follow all the steps in the above testing section

- [ ] Verify that no errors appear in the JS console

### CODE REVIEW

**Review by ctkochan22 — DISMISSED** [review]
Code looks good. Going to pull and test

**Comment by ctkochan22** [comment]
Tested well for me

File: src/pages/signin/SignInPageLayout/index.js
Comment (Santhosh-Sellavel) [inline src/pages/signin/SignInPageLayout/index.js:75]: Unexpected line break

**Review by Santhosh-Sellavel — COMMENTED** [review]
NAB, will test and update.

**Comment by Santhosh-Sellavel** [comment]
mWeb & native UI should be similar but mWeb UI looks broken. 

Screenshot 2022-10-14 at 2 27 08 AM

**Comment by Santhosh-Sellavel** [comment]
Seems like spacing issue on the web 

### on staging
Screenshot 2022-10-14 at 2 28 18 AM

### on this build
Screenshot 2022-10-14 at 2 28 25 AM

**Comment by tgolen** [comment]
Thanks! I have fixed the styles so they are consistent on all platforms now. Just a side note though, these styles don't adjust well with the keyboard avoiding view, primarily on the password step (the keyboard partially covers up the submit button). That's listed as the first test on this PR, and I'm not going to spend time fixing it in this PR, so I am going to remove that as a test and a fixed GH. It'll need to be tackled separately. Here are new screenshots from my recent changes on all platforms:

<details>
<summary>Screenshots of all platforms</summary>







image


</details>

File: src/pages/home/ReportScreen.js
Comment (Santhosh-Sellavel) [inline src/pages/home/ReportScreen.js:231]: Why was this change made?
Why shouldn't we modify the `screenWrapperStyle`  variable instead?

**Comment by Santhosh-Sellavel** [comment]
Tested this works fine https://github.com/Expensify/App/pull/11586#issuecomment-1278172416. But still testing is pending and will take more time, will continue tomorrow!

File: src/pages/home/ReportScreen.js
Comment (tgolen) [reply src/pages/home/ReportScreen.js:231]: It was made to fix the spacing bug you pointed out. I'm not sure about the `screenWrapperStyle`, I was not aware of that prop. Is there a problem with this?

Comment (Santhosh-Sellavel) [reply src/pages/home/ReportScreen.js:231]: Which issue are you referring, I pointed out on Login, it doesn't seem relevant?

Comment (tgolen) [reply src/pages/home/ReportScreen.js:231]: Oh, sorry! I got the components mixed up. I get what you're saying now :D You're right, I've changed this to modify the `screenWrapperStyle` now.

**Review by ctkochan22 — DISMISSED** [review]
Tested well for me. LG

**Comment by Santhosh-Sellavel** [comment]
@tgolen All the issues linked here https://github.com/Expensify/App/issues/10273 is not fixed right, let's remove this from fixed issues.

**Comment by Santhosh-Sellavel** [comment]
@tgolen 

Found the following issues

### iPad issues found in this build
1. Some view is blocking the chat list while the keyboard is active

https://user-images.githubusercontent.com/85645967/196753599-2c45df2f-6b8d-4d85-b7ad-31f8497f5afb.mov

2. Chat List Header User Avatar broken when the keyboard is active

https://user-images.githubusercontent.com/85645967/196763709-eddb6064-407b-4f8f-991d-d61d2e9ff8f4.mov

### Emoji selector on the workspace invite page

#### Fails for android
https://user-images.githubusercontent.com/85645967/196765389-3fdf489e-3ba9-41ae-bf61-d874c431b5ee.mov

**Comment by tgolen** [comment]
Thanks for the testing so far!

I briefly looked into the iPad issue and I can't find any quick and easy fix for it. I tried all the different "behaviors" for the keyboardAvoidingView and they all resulted in the same behavior. I believe it's more of an issue with how the sidebar view is built and styled with flex. I didn't look into it further than that since iPad isn't an officially supported platform yet and not required for testing.

I also looked into that Android issue you pointed out and it is working for me on a pixel 5 emulator. Do you think it's just an android version issue?

image

**Comment by tgolen** [comment]
More update on the iPad issue:

Going back to my tests, I see one of the things that this PR fixed was the placement of the green + button. Indeed, that issue is fixed, but at the cost of this new issue of the LHN kind of being covered up. So, I still think this PR puts the code on a more solid foundation which will allow someone else to come in and clean up the new issue with the LHN being covered up.

I have a sneaky suspicion that the same thing that was previously causing the FAB to move up, is also the thing that is now covering up the LHN.

**Comment by tgolen** [comment]
sorry @ctkochan22, I don't know why it dismissed your earlier review.

**Comment by Santhosh-Sellavel** [comment]
> I also looked into that Android issue you pointed out and it is working for me on a pixel 5 emulator. Do you think it's just an android version issue?

Could be But still, it's an issue right? 

I am using Pixel 4a - Android 13. Which version of android are you using?

**Comment by tgolen** [comment]
image

Can you try it on a pixel 5 and see if you're still able to reproduce it?

**Comment by Santhosh-Sellavel** [comment]
You are using Android 10 its older version. Can you try with 12 or 13? Also android's minimum target SDK 31 which is Android 12

**Comment by Santhosh-Sellavel** [comment]
Here is on pixel 5 running 33 (Android 13)

https://user-images.githubusercontent.com/85645967/197875536-44666623-c227-4542-a901-5602ca022af5.mov

**Comment by tgolen** [comment]
Hm, OK. Let me give that a whirl. Thanks!

**Comment by tgolen** [comment]
OK, yep, you're right. I updated my emulator and SDK and now it's back to not working. What I'm going to do then is just remove the android emoji issue from the list of fixes.

**Comment by tgolen** [comment]
bump @Santhosh-Sellavel this is ready for you again

**Comment by Santhosh-Sellavel** [comment]
Will test today!

**Comment by Santhosh-Sellavel** [comment]
Screen Recordings

#### 🎯 Keyboard covers input on a chat with a draft
<details>

#### Web
Screenshot 2022-10-28 at 3 21 48 AM

#### Desktop
Screenshot 2022-10-28 at 3 21 21 AM

#### Mobile Web - Chrome && Mobile Web - Safari
https://user-images.githubusercontent.com/85645967/198405309-277fad6b-bebd-488e-b20f-f41ffb1ff15e.mov

#### iOS && Android
https://user-images.githubusercontent.com/85645967/198405335-596d0967-5e61-4afe-860d-9d048f402796.mov

</details>

#### 🎯  LHN - The green + button moves
<details>

#### iPad
https://user-images.githubusercontent.com/85645967/198406026-e159ccd5-9535-4333-b761-f9da0f92dfae.mp4

#### iOS &&Android
https://user-images.githubusercontent.com/85645967/198406674-55b0aba0-6394-4d59-8702-c51dc8cd8d73.mov

#### Mobile Web - Chrome &&  Mobile Web - Safari
https://user-images.githubusercontent.com/85645967/198406837-b2440caa-d688-456c-b02f-3390b15c8f6a.mov

#### Web & Desktop
https://user-images.githubusercontent.com/85645967/198405717-41d735d3-2563-43a6-8a3b-06879567c4a9.mov

</details>

**Comment by Santhosh-Sellavel** [comment]
<details>
<summary><h4>PR Reviewer Checklist</h4>

The reviewer will copy/paste it into a new comment and complete it after the author checklist is completed
</summary>

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] iOS / native
    - [x] Android / native
    - [x] iOS / Safari
    - [x] Android / Chrome
    - [x] MacOS / Chrome
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product was added in all `src/languages/*` files
    - [x] I verified any copy / text that was added to the app is correct English and approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

</details>

**Review by Santhosh-Sellavel — APPROVED** [review]
Recordings: https://github.com/Expensify/App/pull/11586#issuecomment-1294145649
Checklist: https://github.com/Expensify/App/pull/11586#issuecomment-1294166229

Based on current Tests, On iPad, a blocking view is added to chat list while the keyboard is active

C+ Reviewed
🎀 👀 🎀

**Comment by Santhosh-Sellavel** [comment]
@tgolen 
Requesting a price increase for this review!

**Comment by tgolen** [comment]
@ctkochan22 Can you please have a final look and merge this?

**Comment by tgolen** [comment]
> Requesting a price increase for this review!

I'm approving a payment of $750 for this 👍

**Comment by Santhosh-Sellavel** [comment]
Thanks 👍 @tgolen

**Review by ctkochan22 — APPROVED** [review]

**Comment by ctkochan22** [comment]
Everything is checked off, i'm going to merge

**Comment by tgolen** [comment]
@jliexpensify For this PR, @Santhosh-Sellavel was the C+ that reviewed this. I believe they normally get paid $250 for reviewing. Because this was such a large PR and process, I have approved for them to be paid $750 instead. Does that make sense?

**Comment by jliexpensify** [comment]
Created job: https://www.upwork.com/ab/applicants/1587571561956552704/job-details

Invited @Santhosh-Sellavel. Just to confirm: are we waiting the requisite 7 days after deployment, or am I ok to pay now?

**Comment by Daniabbasi25** [comment]
> Created job: https://www.upwork.com/ab/applicants/1587571561956552704/job-details
> 
> Invited @Santhosh-Sellavel. Just to confirm: are we waiting the requisite 7 days after deployment, or am I ok to pay now?

Hi, hope you are doing well. I read issue #11586  on GitHub and found that the theKeyboardAvoidingView issue I will resolve it. React Native provide a KeyboardAvoidingView I will rap the view inside this that cause a problem and it will automatically adjust its height position and bottom padding depending upon the keyboard size. It will work on all platforms.

**Comment by tgolen** [comment]
@jliexpensify You can still wait the requisite 7 days.

@Daniabbasi25 Thanks for taking an interest on this issue. We have implemented the `keyboardAvoidingView` everywhere, so we are all good!

**Comment by Santhosh-Sellavel** [comment]
> Created job: https://www.upwork.com/ab/applicants/1587571561956552704/job-details
> 
> Invited @Santhosh-Sellavel. Just to confirm: are we waiting the requisite 7 days after deployment, or am I ok to pay now?

Accepted invite, thanks!

**Comment by jliexpensify** [comment]
Cool, I'll make a note to pay in 2 days!

**Comment by jliexpensify** [comment]
Thanks Rajat - I'll hold off on payment then.

**Comment by Santhosh-Sellavel** [comment]
@jliexpensify 
Here is @tgolen'comment https://github.com/Expensify/App/issues/12480#issuecomment-1317215555

> Hi all, due to the work I did in https://github.com/Expensify/App/pull/11586, which probably forced a regression in this area, please don't hold any regressions against the contributor.

**Comment by jliexpensify** [comment]
Thanks for digging that up @Santhosh-Sellavel , I'll pay now.

**Comment by jliexpensify** [comment]
Paid @Santhosh-Sellavel and job has been closed in Upworks.

File: src/components/FloatingActionButton.js
Comment (tgolen) [inline src/components/FloatingActionButton.js:76]: This was a mistake to add because it created duplicate keyboard avoiding views. This was caught in the issue here: https://github.com/Expensify/App/issues/10905

Comment (tgolen) [reply src/components/FloatingActionButton.js:76]: The other one is in `BaseScreenWrapper`

File: src/components/KeyboardAvoidingView/index.js
Comment (tgolen) [inline src/components/KeyboardAvoidingView/index.js:2]: When this file was removed, it led to a lot of regressions by having Android use `<KeyboardAvoidingView>`. We found in https://github.com/Expensify/App/issues/11087 that it's best to not use `<KeyboardAvoidingView>` on Android at all.

**Pull request merged**
